examples/disclosure/disclosure-navigation-hybrid.html

```diff
@@ -0,0 +1,378 @@
+      <li><a href="../../#disclosure">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+  <h1>Example Links + Disclosure for Navigation Menus</h1>
+  <p>
+    The following example demonstrates using the
+    <a href="../../#disclosure">disclosure design pattern</a>
+    to show and hide dropdown lists of links in a navigation bar for a mythical university web site. Unlike the other <a href="disclosure-navigation.html">disclosure navigation example</a>, this example includes top-level links alongside the disclosure buttons.
+  </p>
+  <p><strong>Note:</strong> Although this example uses the word <q>menu</q> in the colloquial sense to refer to a set of navigation links, it does not use the WAI-ARIA <code>menu</code> role.</p>
+  <p>Similar examples include:</p>
+  <ul>
+    <li><a href="disclosure-navigation.html">Example Disclosure for Navigation Menus</a></li>
+    <li><a href="disclosure-faq.html">Example Disclosure (Show/Hide) for Answers to Frequently Asked Questions</a></li>
+    <li><a href="disclosure-img-long-description.html">Example Disclosure (Show/Hide) for an Image Description</a></li>
+    <li><a href="../menubar/menubar-navigation.html">Navigation Menubar Example</a></li>
+  </ul>
+  <h2>Example Usage Options</h2>
+  <p>
+    This example demonstrates two different ways of implementing keyboard support.
+    Toggle between them with the following checkbox.
+  </p>
+  <label for="arrow-behavior-switch">
+    <input type="checkbox" id="arrow-behavior-switch" checked>
+    Include optional support for arrow keys, <kbd>Home</kbd>, and <kbd>End</kbd>
+  </label>
+
+  <section>
+    <div class="example-header">
+      <h2 id="ex_label">Example</h2>
+    </div>
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of" ></div>
+    <div id="ex1">
+      <nav aria-label="Mythical University">
+        <ul id="exTest" class="disclosure-nav">
+          <li>
+            <a href="#mythical-page-content" class="main-link">About</a>
+            <button type="button" aria-expanded="true" aria-controls="id_about_menu" aria-label="Show About section"> </button>
+            <ul id="id_about_menu">
+              <li>
+                <a href="#mythical-page-content">Overview</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Administration</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Facts</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Campus Tours</a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <a href="#mythical-page-content" class="main-link">Admissions</a>
+            <button type="button" aria-expanded="true" aria-controls="id_admissions_menu" aria-label="Show Admissions section"> </button>
+            <ul id="id_admissions_menu">
```

Comment:
Prevent initial flash.
```suggestion
            <button type="button" aria-expanded="false" aria-controls="id_admissions_menu" aria-label="Show Admissions section"> </button>
            <ul id="id_admissions_menu" style="display: none;">
```